addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -376,7 +376,7 @@
                     </div>
                 </div>
             </div>
-            <div t-else="" t-attf-class="alert my-0 bg-100 p-2 #{'o_wslides_js_channel_enroll' if not is_public_user else ''}"
+            <div t-else="" t-attf-class="alert my-0 bg-100 p-2 #{'o_wslides_js_channel_enroll' if not is_public_user and channel.user_id else ''}"
```

Comment:
Maybe the 'if' should be only around the enroll msg and keep the 'private course' display if no responsible ?

Reply:
@tde-banana-odoo this line removes the `o_wslides_js_channel_enroll` which is the widget that turns the div into a clickable button. Not sure about removing the enroll msg. What if one wants to add an informative message like "send an email to ..."